ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/AddRemoveEditPanel.java

```diff
@@ -0,0 +1,76 @@
+package com.drewhannay.chesscrafter.panel;
+
+import com.drewhannay.chesscrafter.utility.Messages;
+
+import javax.swing.BorderFactory;
+import javax.swing.DefaultListModel;
+import javax.swing.JButton;
+import javax.swing.JList;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.ListCellRenderer;
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+
+public class AddRemoveEditPanel extends JPanel{
```

Comment:
This should probably extend `ChessPanel` and call `super(false)` in the constructor so it won't paint the background